examples/tutorials/advanced/legends.py

```diff
@@ -44,15 +44,18 @@
 
 
 # %%
-# Adjust the position
-# -------------------
+# Adjust position and line spacing
+# --------------------------------
 #
 # Use the ``position`` parameter to adjust the position of the legend. Add an offset via
-# **+o** for the x- and y-directions. Additionally append **+w** to adjust the width
-# of the legend. Note, no box is drawn by default if ``position`` is used.
+# **+o** for the x- and y-directions. Additionally append **+w** to adjust the width of
+# the legend. Note, no box is drawn by default if ``position`` is used. Additionally,
+# the ``position`` parameter allows to adjust the line spacing between the legend
+# entries. Use the **+l** modifier to change the line spacing factor in units of the
```

Comment:
We will have a more Pythonic `Figure.legend` method after https://github.com/GenericMappingTools/pygmt/pull/4046

Reply:
Sounds good. Set this PR to draft.